test/run_test_apache.sh

```diff
@@ -32,6 +32,21 @@ if [ "$#" -gt 3 ] ; then
 else
   SSL="false"
 fi
+
+if [ "$#" -gt 4 ] && [[ $5 == "--local=true" ]] ; then
+  LOCAL=true
+else
+  LOCAL=false
+fi
+
+tests_pattern="[^(--tests=).*]"
+if [ "$#" -gt 5 ] && [[ $6 =~ $tests_pattern ]] ; then
```

Comment:
Same here, can we check the first parameter that conforms `tests_pattern`, not the 6th one?

Reply:
It's doable using this: https://unix.stackexchange.com/questions/331522/how-do-i-parse-optional-arguments-in-a-bash-script-if-no-order-is-given
However it requires all parameters to be named. It would make this PR too big imo, so I would merge this PR first, then switch to named parameters and then add loop to parse it.